site2/docs/client-libraries-java.md

```diff
@@ -107,7 +107,7 @@ String | `authParams` | String represents parameters for the authentication plug
 long|`operationTimeoutMs`|Operation timeout |30000
 long|`statsIntervalSeconds`|Interval between each stats info<br/><br/>Stats is activated with positive `statsInterval`<br/><br/>Set `statsIntervalSeconds` to 1 second at least |60
 int|`numIoThreads`| The number of threads used for handling connections to brokers | 1 
-int|`numListenerThreads`|The number of threads used for handling message listeners | 1 
+int|`numListenerThreads`|The number of threads used for handling message listeners. The listener thread pool is shared across all the consumers and readers using the "listener" model to get messages. For a given consumer, the listener is always invoked from the same thread to ensure ordering. | 1 
```

Comment:
I would add "If you need multiple threads processing single topic, you should create a subscription with type SHARED and create multiple consumers for such subscription. This will of course not ensure ordering"

Reply:
@losiu97 many thanks for your comments. I've incorporated them, please take a look, thanks again.